### PR TITLE
[CBRD-23762]  Misuse the variable for the maximum number of temp cache entries in the temp cache

### PR DESCRIPTION
**Pull request by dev-samin**
http://jira.cubrid.org/browse/CBRD-23762

### Purpose
Change variable in conditional statement
file_manager.c 9296 line 'nfree_entries_max' change to 'ncached_max'

### Implementation
N/A

### Remarks
N/A

### CODE REVIEW

**Review by wrlawodms — APPROVED** [review]
Thank you for your first contribution ! :)
I think it would be better to change the PR title to a more meaningful one.

@hornetmj This issue requires a new Test case.

**Comment by hornetmj** [comment]
@Greenminalee Could you please write a test scenario to verify this? It's not an easy suggestion since you haven't done your learning about file manager yet. To give you a little hint, it seems that you can verify while adjusting the max_entries_in_temp_file_cache system parameter. Give it a try. You may not be able to complete it.

File: src/storage/file_manager.c [hunk 1]
```diff
@@ -9293,7 +9293,7 @@ file_tempcache_put (THREAD_ENTRY * thread_p, FILE_TEMPCACHE_ENTRY * entry)
   /* lock temporary cache */
   file_tempcache_lock ();
 
-  if (file_Tempcache->ncached_not_numerable + file_Tempcache->ncached_numerable < file_Tempcache->nfree_entries_max)
+  if (file_Tempcache->ncached_not_numerable + file_Tempcache->ncached_numerable < file_Tempcache->ncached_max)
```
Comment (hornetmj) [inline src/storage/file_manager.c]:
```suggestion
  if (file_Tempcache->ncached_not_numerable + file_Tempcache->ncached_numerable <= file_Tempcache->ncached_max)
```

**Comment by hornetmj** [comment]
> Thank you for your first contribution ! :) **I think it would be better to change the PR title to a more meaningful one.**
> 
> @hornetmj This issue requires a new Test case.

Please check the comment from @wrlawodms.

**Review by joohok — APPROVED** [review]

File: src/storage/file_manager.c (hunk 1 above)
Comment (dev-samin) [reply src/storage/file_manager.c]: Thank you for the corrections. I didn't think It need to change the sign. I thought It just need to change the variable. Now I know even it seems simple change like the variable change, I should  consider the impact to bring it.

**Comment by dev-samin** [comment]
> 

Thank you for your comment. I will try to make a more meaningful title. It helped me a lot.

**Review by eido5 — APPROVED** [review]

**Comment by dev-samin** [comment]
@joohok I'm sorry for my mistake. I clicked re-review button. I didn't know what it is.  
Could you give approving again.

**Review by joohok — APPROVED** [review]

**Review by joohok — APPROVED** [review]

**Review by hornetmj — APPROVED** [review]

**Comment by hornetmj** [comment]
@mhoh3963 Please merge this PR. The failures of the CircleCI are not related to this PR.

**Pull request merged**